arangod/RocksDBEngine/RocksDBEngine.cpp

```diff
@@ -338,6 +340,30 @@ void RocksDBEngine::shutdownRocksDBInstance() noexcept {
   _db = nullptr;
 }
 
+void RocksDBEngine::flushOpenFilesIfRequired() {
+  if (_metricsLiveWalFiles.load() < _autoFlushMinWalFiles) {
```

Comment:
does the kind of memory order matter in this?

Reply:
Not really. That's why I left it at the default. We could micro-optimize by using acquire and release here, or even relaxed. Then the only consequence would be that flushing may happen a bit earlier or later. But it is not important IMO.